django/forms/renderers/templates.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+from django import forms
+from django.apps import apps
+from django.template import engines
+from django.template.backends.django import DjangoTemplates
+from django.template.backends.jinja2 import Jinja2
```

Comment:
Have you thought about what happens if you don't have jinja2 installed? IMHO jinja2 shouldn't be a hard dependency for Django.
Maybe it should fall back to a DTL based Renderer in that case.